src/store/index.ts

```diff
@@ -53,6 +53,7 @@ const rootReducer = combineReducers({
   [ofacApi.reducerPath]: ofacApi.reducer,
   [safePassApi.reducerPath]: safePassApi.reducer,
   [slices.gatewayApi.reducerPath]: slices.gatewayApi.reducer,
+  [slices.safeOverviewApi.reducerPath]: slices.safeOverviewApi.reducer,
```

Comment:
Wdyt about making this part of the `gatewayApi` slice to keep all related calls in one place? The `SafeOverviewFetcher` could be moved to its own file.